shells/chrome/src/detector.js

```diff
@@ -5,9 +5,14 @@ window.addEventListener('message', e => {
 })
 
 function detect (win) {
+  function findVueRootNode (array, callback) {
+    for (let i = 0; i < array.length; i++) {
+      if (callback.call(array[i])) return array[i]
```

Comment:
No need for the `.call` here either, just `callback(array[i])` would do.

BTW The first argument to `fn.call()` is the `this` context it will be called with.